### ISSUE TITLE
wk1 feb_epic Walk through SAY guide and test in Power BI

### ISSUE THREAD

**Issue by mnallajerla** [lzim/teampsd#2494]
**Session 2 SAY Guide**

- [ ] Need to change information about Data UI vs the way tabs are organized now by VISN 
- [ ] update clinic selections tab instructions if we make 2 (Division and Clinics tabs) 

**Session 3 SAY Guides**

- [ ] Once again, update clinic selections information since it is no longer separate for Data & Viz and Team Data 
- [ ] UPDATE all SP information according to SP tab development and testing

Session 5 Guide Do/Done at bottom GIF: 
https://github.com/mnallajerla/mtl_test/blob/feature-SP_documentation/blue/session05/s05_learner/mtl_session05_see.md

@lijenn  - All the SIM UI GiFs need to be updated in the documentation given the renaming changes

**Comment by mnallajerla** [comment]
**Session 2 SAY Guide**

- [ ] Need to change information about Data UI vs the way tabs are organized now by VISN 

- [ ] update clinic selections tab instructions if we make 2 (Division and Clinics tabs)

**Comment by mnallajerla** [comment]
**Session 3 SAY Guides**

- [ ] Once again, update clinic selections information since it is no longer separate for Data & Viz and Team Data 

- [ ] UPDATE all SP information according to SP tab development and testing

**Comment by mnallajerla** [comment]
HI @lzim @lijenn - 

I went through and tested the Session 2 and 3 SAY guides that are facilitator-facing. There are some minor updates needed, in accordance with the recent changes to PBI, specifically the changes in organization by VISN, the number of clinic selection tabs, updates to SP (see above for full list). 

Once we have finalized these changes, I can go ahead and make edits this next week and have them ready for QA.
